packages/language-server/test/completion-items-classes-spec.ts

```diff
@@ -332,6 +333,50 @@ describe("the UI5 language assistant Code Completion Services - classes", () =>
       expect(getTextInRange(xmlSnippet, suggestion.textEdit?.range)).to.equal(
         ""
       );
+      // Check the namespace is added at the correct position
+      expect(suggestion.additionalTextEdits, "additionalTextEdits").to.be.empty;
+      const tagName = getTagName(suggestion.textEdit);
+      expectExists(tagName, "tag name in suggestion");
+      const ns = tagName.split(":")[0];
+      const attributes = getAttributes(suggestion.textEdit);
+      expect(
+        attributes,
+        `attributes of ${suggestion.textEdit?.newText}`
+      ).to.have.lengthOf(2); // Namespace and tab stop
+      expect(
+        attributes[0],
```

Comment:
do we need to assert on `attributes[1]` too?

Reply:
We can, not sure if it's necessary since we have some tests that already assert on it

Reply:
lets leave it as is, we will get assertions on both attributes for **free* if/when we implement snapshots for completions as well.